migrations/20161104170802-sample-remove-name-index.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+module.exports = {
+  up: function (queryInterface, Sequelize) {
+    queryInterface.removeIndex('Samples', 'SampleName');
+  },
+
+  down: function (queryInterface, Sequelize) {
+    queryInterface.addIndex(
+      'Samples',
+      ['name'],
+      {
```

Comment:
OK

(... although I could argue that with our usage pattern there wouldn't actually be a penalty for indexing name because samples rarely get created or deleted, and when they're updated, the name is typically not updated).